django/template/loader_tags.py

```diff
@@ -249,6 +250,39 @@ def do_block(parser, token):
     return BlockNode(block_name, nodelist)
 
 
+def construct_relative_path(name, relative_name):
+
+    if not any(map(lambda x: relative_name.startswith(x), [
```

Comment:
I think a list comprehension would be more readable: `any(relative_name.startswith(x) for x in [...])`